src/fetchers/bwei.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require('node-fetch')
+const $ = require('cheerio')
```

Comment:
Can we maybe use https://www.npmjs.com/package/node-html-parser as it only has one dependency and we have to install all dependencies every time we run this in the CI